templates/partials/login-base.html.twig

```diff
@@ -0,0 +1,36 @@
+{% set theme_config = attribute(config.themes, config.system.pages.theme) %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% block head %}
+        <meta charset="utf-8" />
+        <title>{% if header.title %}{{ header.title }} | {% endif %}{{ site.title }}</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
+        <link rel="alternate" type="application/atom+xml" href="{{ base_url_absolute}}/feed:atom" title="Atom Feed" />
+        <link rel="alternate" type="application/rss+xml" href="{{ base_url_absolute}}/feed:rss" title="RSS Feed" />
+        <link rel="icon" type="image/png" href="{{ url('theme://images/favicon.png') }}">
+
+        {% block stylesheets %}
+            {% do assets.addCss('theme://css/font-awesome.css',100) %}
+            {% do assets.addCss('theme://css/font-awesome.min.css',100) %}
+
+            {{ assets.css() }}
+        {% endblock %}
+
+        {% block javascripts %}
+            {{ assets.js() }}
+        {% endblock %}
+
+    {% endblock %}
+</head>
+
+<body>
+{% block content %}
+{% endblock %}
+{% block analytics %}
+    {% if theme_config.google_analytics_code %}
```

Comment:
I personally don't think that the analytics block section is necessary.

I'll pull your changes down and see if it grows on me as I'm used to the "theme dependent" login screen.